main.go

```diff
@@ -21,9 +24,32 @@ func main() {
 	}
 }
 
+func loadTLSCert(l *slog.Logger, certPath, keyPath string) func(*tls.Config) {
+	getCertificate := func(_ *tls.ClientHelloInfo) (*tls.Certificate, error) {
+		cert, err := tls.LoadX509KeyPair(certPath, keyPath)
+		if err != nil {
+			return nil, errors.Wrap(err, "Unable to load TLS certificates")
+		}
+
+		return &cert, err
+	}
+
+	return func(config *tls.Config) {
+		config.GetCertificate = getCertificate
+	}
+}
+
 func run(l *slog.Logger) error {
 	log.SetLogger(logr.FromSlogHandler(l.Handler()))
 
+	var enableTLS bool
+	var tlsCertificatePath string
+	var tlsCertificateKeyPath string
+	flag.BoolVar(&enableTLS, "enable-tls", true, "Toggle tls enablement")
+	flag.StringVar(&tlsCertificatePath, "cert-path", "", "Path to TLS certificate store.")
+	flag.StringVar(&tlsCertificateKeyPath, "key-path", "", "Path to TLS private key.")
```

Comment:
```suggestion
	flag.StringVar(&tlsCertificatePath, "cert-path", "", "Path to TLS certificate store")
	flag.StringVar(&tlsCertificateKeyPath, "key-path", "", "Path to TLS private key")
```

Reply:
The convention I've seen on previous projects is to leave these as full sentences.  As an example, look at the options that the workspaces operator takes.